client/src/components/Rooms/RoomCard/RoomCard.js

```diff
@@ -1,31 +1,36 @@
 import React from "react";
 
-import {Grid} from "@material-ui/core";
-import StarRating from './../../UI/StarRating/StarRating';
+import { Grid } from "@material-ui/core";
+import StarRating from "./../../UI/StarRating/StarRating";
 
 const roomCard = props => {
   const backgroundImage = {
     backgroundImage: `url(${props.background})`
   };
 
   return (
-    <Grid item xs={12} sm={6} md={4} 
-      className="room-card" 
-      style={backgroundImage}
-      onClick={props.clicked}>
-      <Grid className="room-card-desc">
-        <Grid className="cardInfo">
-          <div>{props.category}</div>
-          <div>
-            {props.members} / {props.limit}
+    <Grid item xs={12} sm={6} md={4} className="col" ontouchstart="this.classList.toggle('hover');">
+      <div className="room-card" onClick={props.clicked}>
+        <div
+          className="front"
+          style={backgroundImage}
+        >
+          <img src={props.avatar} alt={props.avatar}/>
```

Comment:
Why avatar link in alt attribute? Does this give any benefit?

Reply:
No, it doesn't...I use {props.avatar} like unique props for alt...we don't use it, it just for fixing alarm in console: "img elements must have an alt prop...."